### PR TITLE
fix(@ngtools/webpack): change of CSS no longer breaks rebuild

### LINKED ISSUE #4326: Webpack rebuild error
### OS?
Ubuntu 16.04

### Versions
angular-cli: 1.0.0-beta.28.3
node: 7.4.0
os: linux x64
@angular/common: 2.4.5
@angular/compiler: 2.4.5
@angular/core: 2.4.5
@angular/forms: 2.4.5
@angular/http: 2.4.5
@angular/platform-browser: 2.4.5
@angular/platform-browser-dynamic: 2.4.5
@angular/router: 3.4.5
@angular/compiler-cli: 2.4.5

### Repro steps.
Created the app with the `angular-cli` (`ng new`).  Ran `ng serve` to make sure everything was working.  All was good, webpack worked and reported:

    Hash: 7a95e8c73d53b9f210c7                    
    Time: 8041ms
    chunk    {0} main.bundle.js, main.bundle.map (main) 17.2 kB {3} [initial] [rendered]
    chunk    {1} styles.bundle.js, styles.bundle.map (styles) 205 kB {4} [initial] [rendered]
    chunk    {2} scripts.bundle.js, scripts.bundle.map (scripts) 375 kB {4} [initial] [rendered]
    chunk    {3} vendor.bundle.js, vendor.bundle.map (vendor) 2.83 MB [initial] [rendered]
    chunk    {4} inline.bundle.js, inline.bundle.map (inline) 0 bytes [entry] [rendered]
    webpack: Compiled successfully.

I was able to load the app in a browser.  Made a small change to the `app.component.ts` file (removed the trailing new line).  Saved the file, webpack attempted to re-bundle the file, but failed.

### The log given by the failure.
    Hash: 7a95e8c73d53b9f210c7                
    Time: 613ms
    chunk    {0} main.bundle.js, main.bundle.map (main) 17.2 kB {3} [initial]
    chunk    {1} styles.bundle.js, styles.bundle.map (styles) 205 kB {4} [initial]
    chunk    {2} scripts.bundle.js, scripts.bundle.map (scripts) 375 kB {4} [initial]
    chunk    {3} vendor.bundle.js, vendor.bundle.map (vendor) 2.83 MB [initial]
    chunk    {4} inline.bundle.js, inline.bundle.map (inline) 0 bytes [entry]

    ERROR in Cannot read property 'length' of undefined
    webpack: Failed to compile.

### LINKED ISSUE #4329: Browser does not refresh after code changed
angular-cli: 1.0.0-beta.28.3
node: 6.9.2
os: win32 x64
@angular/common: 2.4.5
@angular/compiler: 2.4.5
@angular/core: 2.4.5
@angular/forms: 2.4.5
@angular/http: 2.4.5
@angular/platform-browser: 2.4.5
@angular/platform-browser-dynamic: 2.4.5
@angular/router: 3.4.5
@angular/compiler-cli: 2.4.5

`ng serve`
code change
`
webpack: Compiling...
Hash: 228604d16756d5a0fdd3
Time: 717ms
chunk    {0} polyfills.bundle.js, polyfills.bundle.map (polyfills) 222 kB {4} [initial]
chunk    {1} main.bundle.js, main.bundle.map (main) 3.97 kB {3} [initial]
chunk    {2} styles.bundle.js, styles.bundle.map (styles) 10 kB {4} [initial]
chunk    {3} vendor.bundle.js, vendor.bundle.map (vendor) 2.64 MB [initial]
chunk    {4} inline.bundle.js, inline.bundle.map (inline) 0 bytes [entry]
webpack: Compiled successfully.
`

browser don't refresh. <F5> do not solve the problem
rerun `ng serve` => change available

### PR DESCRIPTION
**Pull request by Brocco**
Fixes #4326
Fixes #4329

### CODE REVIEW

File: tests/e2e/tests/build/rebuild-css-change.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+} from '../../utils/process';
+import {appendToFile} from '../../utils/fs';
+
+
+export default function() {
+  if (process.platform.startsWith('win')) {
+    return Promise.resolve();
+  }
+
+  return silentExecAndWaitForOutputToMatch('ng', ['serve'], /webpack: bundle is now VALID|webpack: Compiled successfully./)
```
Comment (hansl) [inline tests/e2e/tests/build/rebuild-css-change.ts]: This line seems awfully long... is it more than 100 characters?

**Comment by Brocco** [comment]
nit fixed using regex consts

**Comment by beeman** [comment]
@Brocco tested the fix in the PR, does not trigger the error anymore 👍

**Comment by born2net** [comment]
great guys, when do you think this will get into NPM?

**Pull request merged**